cylc/flow.cylc

```diff
@@ -0,0 +1,57 @@
+[scheduler]
+    allow implicit tasks = True
+[scheduling]
+    cycling mode = integer
+    initial cycle point = 1
+    [[graph]]
+        R1 = """
+            mkpaths => fetchdata => soit #& landmask => preprocess => extractfeatures => tracking
+             """
+[runtime]
+    [[root]]
+        [[[environment]]]
+            # Update these variables with your run parameters
+            startdate = "2022-05-04"
+            enddate = "2022-05-06"
+            crs = "wgs84" #epsg3413 for polar stereographic
+            bounding_box = "78.186394 38.250605 70.749318 15.32373"
+            centroid_x = "80"
+            centroid_y = "-18"
+            minfloearea = "300"
+            maxfloearea = "90000"
+            # Recommend using these default paths for output
+            project_dir = "~/ice-floe-tracker-pipeline"
+            fetchdata_dir = $project_dir/"images"
+            truecolor_dir = "images/truecolor"
+            reflectance_dir = "images/reflectance"
+            landmask_dir = "output/landmasks"
+            preprocess_dir = "output/preprocess"
+            soit_dir = "output/soit"
+            features_dir = "output/features"
```

Comment:
As you can see the pipeline repo already has a nice folder structure to organize the files. 
```bash
.
├───config
├───resources
├───results
├───src
├───test
│   └───test_inputs
│       ├───expected
│       ├───input_pipeline
│       ├───pipeline
│       │   └───feature_extraction
│       │       └───input
│       └───tracker
└───workflow
    ├───report
    └───scripts
```

It might be a good idea to suggest using these folders within this structure. For example,
```
fetchdata_dir = $project_dir/"resources/images" # I am not sure how to put the path together correctly! 
```
The output folder could be `/results`, and so on.